### ISSUE TITLE
all accessibility tests are failing on 3.1.31 - latest Rule Archive.

### ISSUE THREAD

**Issue by jitendra19** [IBMa/equal-access#975]
before release of 3.1.31, things are working on latest ruleArchive.  But this weekend I found out one of violation is getting reported.
Element :  `<span tabindex="0" role="none">`

Rule Id: "element_tabbable_role_valid"

I have achecker.json in root directory with below configurations.

`
{
    "ruleArchive": "04May2022",
    "policies": ["IBM_Accessibility"],
    "failLevels": ["violation", "potentialviolation"],
    "reportLevels": [
        "violation",
        "potentialviolation",
        "recommendation",
        "potentialrecommendation",
        "manual",
        "pass"
    ],
    "outputFormat": ["json"],
    "outputFolder": "results",
    "baselineFolder": "baselines"
}

`

But it is going to **latest** (https://able.ibm.com/rules/archives/latest/js/ace.js) , ignoring config's ruleArchive value (04May2022).

**Comment by eirckr** [comment]
Please update to the latest version 3.1.32, we pushed a hotfix for this issue. Thanks

**Comment by jitendra19** [comment]
@eirckr @shunguoy, Thanks for response. 

I am using "cypress-accessibility-checker": "3.1.32"

below is configuration in file **.achecker.yml**

`
ruleArchive : **04May2022**
policies:
    - IBM_Accessibility
    # - IBM_Accessibility_WCAG21
failLevels:
    - violation
    - potentialviolation
reportLevels:
    - violation
    - potentialviolation
    - recommendation
    - potentialrecommendation
    - manual
outputFormat: json
label:
    - master
outputFolder: results
baselineFolder: baselines
checkHiddenContent: false
`

But still this is going to validate on -  https://cdn.jsdelivr.net/npm/accessibility-checker-engine@3.1.31/ace.js 

my question is why this is not taking ruleArchive : **04May2022**

I tried to see internal of cypress-accessibility-checker, but there is nothing which can replace with @latest

Please guide us to resolve this.
